### ISSUE TITLE
Autocomplete with completions from standard library wont work

### ISSUE THREAD

**Issue by odinase** [yasuyuky/autocomplete-clang#160]
Hey,

I have been trying to make the autocomplete function work when I for example use an instance of a standard library class, say vector. When I type the name of the instance, I expect to get suggestions for member functions of the vector class when I type a dot after the name. I have no idea where the error is and would very much like som input on where to look. 

![image](https://user-images.githubusercontent.com/31511833/29929931-3b59a054-8e6d-11e7-8b33-826d282f5826.png)

I do get a drop-down menu, but it only contains keywords that have already appeared in one of the files in the the project.

Basically, what I am trying to achieve is the drop-down menu that appears in the gif at the top of the autocomplete-clang "home page" (https://atom.io/packages/autocomplete-clang).

**Comment by tiferrei** [comment]
Ditto.
